staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -35,12 +35,17 @@ import (
 
 var (
 	explainLong = templates.LongDesc(i18n.T(`
-		Describe fields and structure of various resources.
+		Describe (sub-)fields and structure of various resources.
 
-		This command describes the fields associated with each supported API resource.
-		Fields are identified via a simple JSONPath identifier:
+		This command describes the (sub-)fields associated with each supported API resource.
+		Fields are identified via a simple JSONPath accessor:
 
-			<type>.<fieldName>[.<fieldName>]
+			<type>.<fieldName>
+
+		Subfield of each field is accessed through the same dotted notation:
+
+			<type>.<fieldName>[.<subfieldName>]
+			<type>.<fieldName>.<subfieldName>[.<subfieldName>]
```

Comment:
Because the help text references JSONPath, which is a well known query language, this feels unnecessarily verbose to me, especially when paired with the example section that displays how to access subfields for the user.